src/VisualStudio/Core/Test/Venus/DocumentServiceTests.vb

```diff
@@ -100,6 +100,26 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Venus
             End Using
         End Sub
 
+        <Fact, Trait(Traits.Feature, Traits.Features.Workspace)>
+        Public Sub TestSourceGeneratedDocumentOperation()
```

Comment:
It's very odd this is in the "Venus" folder -- do we have tests anywhere else for these or just here?

Reply:
(the test itself looks fine)

Reply:
This folder contains the only tests for this.